src/cplscheme/BaseCouplingScheme.hpp

```diff
@@ -225,20 +209,32 @@ class BaseCouplingScheme : public CouplingScheme {
   void setIterationAcceleration(acceleration::PtrAcceleration acceleration);
```

Comment:
We cannot remove it, since the function is used by `CouplingSchemeConfiguration::createSerialImplicitCouplingScheme`, `CouplingSchemeConfiguration::createParallelImplicitCouplingScheme` and `CouplingSchemeConfiguration::createMultiCouplingScheme`. Any name suggestions?